README.md

```diff
@@ -273,6 +273,11 @@ charts local path `charts/starship`.
 ```shell
 git clone git@github.com:tricorder-observability/helm-charts.git
 cd helm-charts
+
+# Add dependency repos
```

Comment:
This only needs when we are in development for the helm chart. Because we already packaged those dep charts in the CI stage.

Reply:
So, It's better to add this to `development` section?